src/middlewared/middlewared/plugins/reporting/events.py

```diff
@@ -31,19 +31,11 @@ class RealtimeEventSource(EventSource):
         Dict('interfaces', additional_attrs=True),
         Dict(
             'memory',
-            Dict(
-                'classes',
-                Int('apps'),
-                Int('arc'),
-                Int('buffers'),
-                Int('cache'),
-                Int('page_tables'),
-                Int('slab_cache'),
-                Int('unused'),
-            ),
-            Dict('extra', additional_attrs=True),
+            Int('arc_size'),
+            Int('arc_free_memory'),
+            Int('arc_available_memory'),
+            Int('physical_memory_total'),
         ),
-        Dict('virtual_memory', additional_attrs=True),
         Dict(
             'zfs',
             Int('arc_free_memory'),
```

Comment:
I want the `arc_free_memory`, `arc_size`, and `arc_available_memory` removed from the `zfs` key. Otherwise, we have duplicate fields being returned for 2 different top-level keys. The idea is to keep "memory" related statistics to the `memory` key and all other zfs related statistics to the `zfs` key.